src/components/Header/index.js

```diff
@@ -15,37 +15,37 @@ function Header() {
     setOpen(false);
   }
 
-
+  
 
   return (
-    <nav className='navbar'>
+    <Container className='navbar'>
       <Link to='/' className='nav-logo'>
         <img
           src={logo}
           alt='Mozdevz'
         />
       </Link>
-      <div onClick={handleClick} className='nav-icon'>
+      <NavIcon onClick={handleClick}>
         {open ? <FiX /> : <FiMenu />}
-      </div>
-      <ul className={open ? 'nav-links active' : 'nav-links'}>
+      </NavIcon>
+      <NavList active={open} >
         <li className='nav-item'>
-          <Link to='/' className='nav-link' onClick={closeMenu}>
+          <NavLink to='/' onClick={closeMenu}>
             Inicio
-          </Link>
+          </NavLink>
         </li>
         <li className='nav-item'>
-          <Link to='/programas' className='nav-link' onClick={closeMenu}>
+          <NavLink to='/programas' onClick={closeMenu}>
             Programas
-          </Link>
+          </NavLink>
         </li>
         <li className='nav-item'>
-          <Link to='/contacto' className='nav-link' onClick={closeMenu}>
+          <NavLink to='/contacto' className='nav-link' onClick={closeMenu}>
```

Comment:
Why did you keep the `className='nav-link'` on this line??? Isnt the `<NavLink>` a styled component already??

Reply:
Ups. I forgot to remove, It shouldn't be there